docs/flow.mdx

```diff
@@ -18,14 +18,20 @@ alternatives:
 import styled from '@emotion/styled'
 
 // Option A
-const A = styled<Props>('div')`
+const A =
```

Comment:
this has been broken - since we haven't dropped Flow on the main branch just yet... would you be willing to fix this?

Reply:
Maybe it would be enough to just revert this single file? I don't think we run Prettier automatically over docs files so it should pass the CI

Reply:
Ah, I thought I'd excluded `.md` & `.mdx` files, sloppy -- reverted them in a6a05c11!